src/schema/v2/me/update_me_mutation.ts

```diff
@@ -123,13 +153,32 @@ export default mutationWithClientMutationId<any, any, ResolverContext>({
     me: Me,
   },
   mutateAndGetPayload: (
-    { collectorLevel, priceRangeMin, priceRangeMax, ..._user },
+    {
+      collectorLevel,
+      priceRangeMin,
+      priceRangeMax,
+      receivePurchaseNotification,
+      receiveOutbidNotification,
+      receiveLotOpeningSoonNotification,
+      receiveSaleOpeningClosingNotification,
+      receiveNewWorksNotification,
+      receiveNewSalesNotification,
+      receivePromotionNotification,
+      ..._user
+    },
     { updateMeLoader }
   ) => {
     const user: any = {
       collector_level: collectorLevel,
       price_range_min: priceRangeMin,
       price_range_max: priceRangeMax,
+      receive_purchase_notification: receivePurchaseNotification,
+      receive_outbid_notification: receiveOutbidNotification,
+      receive_lot_opening_soon_notification: receiveLotOpeningSoonNotification,
+      receive_sale_opening_closing_notification: receiveSaleOpeningClosingNotification,
+      receive_new_works_notification: receiveNewWorksNotification,
+      receive_new_sales_notification: receiveNewSalesNotification,
+      receive_promotion_notification: receivePromotionNotification,
```

Comment:
I just want to verify my understanding here, based on Joey's comment in the Gravity PR. So because we're passing these through to Gravity, and they default to `undefined` in Metaphysics (because they are optional), then Gravity won't change anything unless these are specified in the mutation to Metaphysics. In other words, if these aren't included in the input to the mutation, then they won't change anything in Gravity, right?

Reply:
Good call out, let me manually test and see if I can add a test in here or gravity for that behavior

Reply:
I tested manually and not specifying one of the optional params in mutation does pass through the param as undefined to the gravity loader but when path is generated they are ignored so this does not modify the param on gravities side as expected. Added a test for this behavior in `toKey` helper function used for generating paths.